Add issue templates (#14)
kyma-incubator/reconciler: .github/ISSUE_TEMPLATE/documentation-improvement.md

---
name: Documentation improvement
about: Suggest an improvement or report a bug in the documentation
---

<!-- Thank you for your contribution. Before you submit the issue:
1. Search open and closed issues for duplicates.
2. Read the contributing guidelines.
3. Assign the Documentation project.
-->

**Description**

<!-- Provide a clear and concise description of the potential documentation improvement.-->

**Area**

<!-- Provide the area the document refers to. For example, write: 
* Application Connector
* Event Mesh
* Kyma Environment Broker
* Rafter -->

**Reasons**

> <!-- Explain why we should improve the document. -->

Reviewer: ```suggestion
<!-- Explain why this would improve the document. -->
```

I think we're not generally opposed to the idea of improving the docs :)

**Assignees**

@kyma-project/technical-writers

**Attachments**

> <!-- Attach any files, links, code samples, or screenshots that will convince us to your idea. -->

Reviewer: ```suggestion
<!-- Attach any files, links, code samples, or screenshots that support your suggestion. -->
```
